dnf-behave-tests/features/microdnf/install.feature

```diff
@@ -0,0 +1,86 @@
+@no_installroot
+Feature: microdnf install command on packages with xml:base
+
+
+Background: Using repositories dnf-ci-fedora and dnf-ci-thirdparty-updates
+  Given I delete file "/etc/yum.repos.d/*.repo" with globs
+
+
+@bz1734350
```

Comment:
Can you add `@bz1779757` here? It's the same crash as the `--nodocs` ones, just this one wasn't reported wtih the `--nodocs` option, so let's tag the simple test with it... Thanks!